drivers/ble/sphero/sphero_r2d2_driver.go

```diff
@@ -0,0 +1,396 @@
+package sphero
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"log"
+	"sync"
+	"time"
+
+	"gobot.io/x/gobot/v2"
+	"gobot.io/x/gobot/v2/drivers/ble"
+	"gobot.io/x/gobot/v2/drivers/common/spherocommon"
+)
+
+type Playback uint8
+
+type LegAction uint8
+
+const (
+	r2AntiDosChara  = "00020005574f4f2053706865726f2121" // handshake 1st transmission
+	r2CommandsChara = "00010002574f4f2053706865726f2121" // send uuid
+	r2ResponseChara = r2CommandsChara
+
+	// command safe interval
+	commandInterval = time.Duration(12) * time.Millisecond
```

Comment:
```suggestion
	commandInterval = 12 * time.Millisecond
```
should normally work